### PR TITLE
Run workflow jobs toghether

### PR DESCRIPTION
**Pull request by mahor1221**
* Making release bundles and packages in parallel which reduces workflow build time and makes debugging each separate job easier.
* Added checksum generation to publish.yml

Test run: https://github.com/mahor1221/Tachidesk-Server/actions/runs/2596937668

### CODE REVIEW

File: .github/workflows/build_push.yml [hunk 1]
```diff
@@ -59,41 +57,139 @@ jobs:
...
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [
+          linux-x64,
+          linux-all,
+          macOS-x64,
+          macOS-arm64,
+          # takes about 6m to build deb package
+          #debian-all,
```
Comment (Syer10) [inline .github/workflows/build_push.yml]: I think it would be fine if this was enabled for previews, it doesnt really matter how long a workflow takes

Comment (Robonau) [reply .github/workflows/build_push.yml]: do u mean enabled for releases? previews being fast is kinda useful sometimes

Comment (Syer10) [reply .github/workflows/build_push.yml]: Tachidesk isnt making builds like no tomorrow, so a little delay in previews isnt really important enough that we have to disable the creation of deb packages.

Comment (mahor1221) [reply .github/workflows/build_push.yml]: You are right, someone can just comment out unwanted parts on their fork.

**Pull request merged**